complete/src/main/java/com/example/restservice/service/smsService/SmsManager.java

```diff
@@ -1,18 +1,22 @@
 package com.example.restservice.service.smsService;
 
+import com.example.restservice.config.PropertiesConfig;
 import lombok.RequiredArgsConstructor;
 import org.springframework.stereotype.Component;
 
 @Component
 @RequiredArgsConstructor
 public class SmsManager {
   private final SmsService smsService;
+  private final PropertiesConfig propertiesConfig;
```

Comment:
while this would work, can you avoid this, and try using @Value anotation?

https://www.baeldung.com/spring-value-annotation

The reason is, we want to have consistency throughout. If we are going the `ConfigurationProperties` route, we should use that globally. 

For a usage example within our repo, refer to this: https://github.com/SimplQ/simplQ-backend/blob/master/complete/src/main/java/com/example/restservice/controller/advices/AuthenticationFilter.java#L36

You would just have to create a private boolean field and annotate it, then use it.